ui/perfherder/compare/RetriggerModal.jsx

```diff
@@ -0,0 +1,223 @@
+        ? baseRetriggerTimes
+        : parseInt(baseRetriggerTimes, 10),
+      newRetriggerTimes: Number.isInteger(newRetriggerTimes)
+        ? newRetriggerTimes
+        : parseInt(newRetriggerTimes, 10),
+    });
+  };
+
+  render() {
+    const {
+      showModal,
+      toggle,
+      isBaseAggregate,
+      maxRetriggersValue,
+      currentRetriggerRow,
+    } = this.props;
+    const { invalidInput } = this.state;
+
+    return (
+      <Modal
+        isOpen={showModal}
+        onOpened={this.onOpened}
+        onClosed={this.onClosed}
+      >
+        <ModalHeader toggle={toggle}>Retrigger Jobs</ModalHeader>
+        <Form>
+          <ModalBody>
+            <div className="row">
+              <FormGroup
+                className="col-xs-10 col-sm-6 col-md-6 col-lg-6 form-inline"
+                inline
+              >
+                <InputGroup title={this.getInputTitle(true)}>
+                  <InputGroupAddon addonType="prepend">
+                    <InputGroupText>Base revision:</InputGroupText>
+                  </InputGroupAddon>
+                  <Input
+                    data-testid="input baseRetriggerTimes"
+                    defaultValue={this.getInitialValue(true)}
+                    min={0}
+                    max={maxRetriggersValue}
+                    onChange={this.handleChange}
+                    name="baseRetriggerTimes"
+                    type="number"
+                    disabled={
+                      isBaseAggregate ||
+                      !currentRetriggerRow.originalRetriggerableJobId
+                    }
+                  />
+                </InputGroup>
+              </FormGroup>
+              <FormGroup
+                className="col-xs-10 col-sm-6 col-md-6 col-lg-6 form-inline"
+                inline
+              >
+                <InputGroup title={this.getInputTitle()}>
+                  <InputGroupAddon addonType="prepend">
+                    <InputGroupText>New revision:</InputGroupText>
+                  </InputGroupAddon>
+                  <Input
+                    data-testid="input newRetriggerTimes"
+                    defaultValue={this.getInitialValue()}
+                    min={0}
+                    max={maxRetriggersValue}
+                    onChange={this.handleChange}
+                    name="newRetriggerTimes"
+                    type="number"
+                    disabled={!currentRetriggerRow.newRetriggerableJobId}
+                  />
+                </InputGroup>
+              </FormGroup>
+            </div>
+            <div className="text-center">
+              {invalidInput && (
+                <p className="text-danger pt-2 text-wrap">
+                  *Inputs should be numbers in 0 - {maxRetriggersValue} range
+                </p>
+              )}
+            </div>
+          </ModalBody>
+          <ModalFooter>
+            <Button
+              color="primary"
```

Comment:
Can you use "info" or "secondary". We don't really use the primary color at all in Perfherder :)

Reply:
It fits a lot better now that I changed it to `info`. Thanks